test/vidom/runtime-test.js

```diff
@@ -0,0 +1,716 @@
+var assert = require('assert');
+var fixtures = require('../fixtures')('vidom');
+var bemxjst = require('../../').vidom;
+
+var test = fixtures.test;
+
+describe('VIDOM compiler/Runtime', function() {
+
+  it('should mix className property if passed', function() {
+    test(function() {
+      block('b1')
+    }, { block: 'b1', attrs: { className: 'hi-there' }},
+      [ 'div', { className: 'b1 hi-there' } ]);
+  });
+
+  // FIXME modify this test to check cases with
```

Comment:
I have moved engine from this pr to own repo, so issues could be pointed there
https://github.com/Guria/xjst-vidom/issues